site/_apps/zuix-hn.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 2
+title: 'Zuix HN'
+github-title: 'genielabs/zuix-hn'
+library: 'ZUIX'
+libraries: 
+    - name: 'ZUIX'
+module-bundling: 'zuix-bundler'
+service-worker: 'Application Shell'
```

Comment:
The service worker is just implement simple caching of app files. Network requests are not being cached in this very moment. But now I see that Workbox can come handy for implementing caching in the sw. I will use that later. Thanks for the hint.